src/models/notifications/notifications.service.ts

```diff
@@ -20,7 +23,37 @@ export class NotificationsService {
     notificationData.deliveryStatus = DeliveryStatus.PENDING;
     const notification = this.notificationRepository.create(notificationData);
     const result = await this.notificationRepository.save(notification);
-    await this.notificationQueueService.addNotificationToQueue(notification);
     return result;
   }
+
+  @Cron(CronExpression.EVERY_MINUTE)
+  async addNotificationsToQueue(): Promise<void> {
```

Comment:
Add TODO

I think we should move this to a different file.

Reply:
This has been added in latest commit